docs/content/en/docs/Concepts/mqtt/index.md

```diff
@@ -539,11 +622,26 @@ Shopfloor insights are recommendations for action that require concrete and rapi
 ```
 {{%/alert%}}
 
+## Explanation of IDs
+
+There are various different IDs that you can find in the MQTT messages. This section is designed to give an overview.
+
+
+| Name | data type/format | description |
+|-----|-----|--------------------------|
+| [`product_id`](/docs/concepts/mqtt/#explanation-of-ids) | int | The type of product that should be produced in an order for a specific asset. Can be used to retrieve the target speed. |
+| [`order_id`](/docs/concepts/mqtt/#explanation-of-ids) | int | Order ID, which provides the type of product (see [`product_id`](/docs/concepts/mqtt/#explanation-of-ids)) and the amount of pieces that should be produced. |
+| [`uniqeProductAlternativeID`](/docs/concepts/mqtt/#explanation-of-ids) | int | In short: AID. Used to describe a single product of the type [`product_id`](/docs/concepts/mqtt/#explanation-of-ids) in the order [`order_id`](/docs/concepts/mqtt/#explanation-of-ids). This is the ID that might be written on a physical on the product and is usually the relevant ID for engineers and for production planning. It usually stays the same. |
```

Comment:
This is the ID that might be written on the product (with a physical Label, lasered on to it etc.: technologie independant) and is usually the relevant...